packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthrough-types/aPIGateway-user-input-types.ts

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
naming of this file seems improper

Reply:
It's intentional, it's the service name 'API Gateway' with no spaces and the first character to lower case, because that's what the cli-core code is looking for